VMBackup/main/PluginHost.py

```diff
@@ -199,8 +201,8 @@ def pre_script(self):
             curr = curr + 1
 
         flag = True
-        for i in range(0,((self.timeoutInSeconds)/5)+2): #waiting 10 more seconds to escape race condition between Host and script timing out
-            time.sleep(5)
+        for i in range(0, int(self.timeoutInSeconds/self.pollTime) + 2): #waiting 10 more seconds to escape race condition between Host and script timing out
```

Comment:
make sure this syntax works in all like pyrthon 2.6
int(self.timeoutInSeconds/self.pollTime)

Reply:
both are made into int right...what exactly should i test ?